aws/aws.go

```diff
@@ -454,9 +455,37 @@ func deleteVolumes(ec2Clients map[string]ec2Client, volumes []*types.Disk) []err
 				if ctx.DryRun {
 					log.Infof("[AWS] Dry-run set, volume is not deleted: %s:%s, region: %s", vol.Name, vol.ID, region)
 				} else {
-					log.Infof("[AWS] Delete volume: %s:%s", vol.Name, vol.ID)
-					if _, err := ec2Client.DeleteVolume(&ec2.DeleteVolumeInput{VolumeId: &vol.ID}); err != nil {
-						errChan <- err
+					log.Infof("[AWS] Initiate delete volume: %s:%s", vol.Name, vol.ID)
```

Comment:
An early return of `ctx.DryRun` without an else moves entire function to the left.

Reply:
I followed the pattern what we already have here in this file.